### PR TITLE
Add xp to monsters

### PR DESCRIPTION
**Pull request by ragnork2**
## What does this do?
Adds an xp field to monsters

## How was it tested?
VScode linter

## Is there a Github issue this is resolving?
Issue #299 

## Did you update the docs in the API? Please link an associated PR if applicable.
I did not, not sure if i needed to, lmk if i do
## Here's a fun image for your troubles
random photo - update me

### CODE REVIEW

File: src/5e-SRD-Monsters.json [hunk 1]
```diff
@@ -40348,4 +40670,4 @@
     ],
     "url": "/api/monsters/zombie"
   }
-]
+]
```
Comment (bagelbits) [inline src/5e-SRD-Monsters.json:40673]:
```suggestion
]

```

**Review by bagelbits — APPROVED** [review]
Yeah! Nice!

File: src/5e-SRD-Monsters.json (hunk 1 above)
Comment (ragnork2) [reply src/5e-SRD-Monsters.json:40673]: I have no idea why VScode decided to do that lol

Comment (bagelbits) [reply src/5e-SRD-Monsters.json:40673]: It's probably a setting somewhere?

Comment (fergcb) [reply src/5e-SRD-Monsters.json:40673]: @ragnork2 The setting in VS code is called "insert final newline". You should be able to search for it in the settings menu.

Comment (ragnork2) [reply src/5e-SRD-Monsters.json:40673]: ah kk thanks!

**Comment by ogregoire** [comment]
Since the CR and the XP are strongly related, it might have been a good idea to have made a structure of the two, like:

```
{
  challenge: {
    rating: "0",
    xp: 0
  }
}
```

**Comment by bagelbits** [comment]
@ogregoire You raise a valid point.

**Comment by fergcb** [comment]
I can't say I'm a fan of that idea.

It complicates the structure with no real benefit for the client, as far as I can see; this structure isn't exactly reusable, and having the properties belong directly to the monster isn't exactly confusing.

It also goes against the pattern of query parameters having the same names as the attribute they correspond to in the document.

If we *were* to make this change, I don't think `challenge` is a suitable name. Sure, XP comes from CR, but it is not *part* of the challenge rating itself. That said, I can't think of anything more appropriate...

**Comment by ogregoire** [comment]
Yes, "Challenge" is the name. See SRD p. 258.

**Comment by fergcb** [comment]
"Challenge" refers to the challenge rating. The XP in brackets is not a component of the "challenge", but a result of it. 

I read "**Challenge** 2 (450xp)" not as `challenge: { rating: 2, xp:450 }`, but as `challenge: 2, (xp: 450)`.

"Challenge" is an appropriate label for the challenge rating, but I don't feel that label reflects the fact that XP is included in that property.

**Pull request merged**